workflow/transport/create_road_network.py

```diff
@@ -83,22 +85,26 @@ def get_road_condition(row: pd.Series) -> Tuple[str, str]:
         row: Must have surface (nullable) and highway attributes.
 
     Returns:
-        Boolean paved status and surface category string
+        surface category string
     """
-
     if not row.tag_surface:
         if row.tag_highway in {"motorway", "trunk", "primary"}:
-            return True, "asphalt"
+            return "asphalt"
```

Comment:
Perhaps it's not worth trying to gap fill the material by guessing from the highway classification. We could strip this bit of guesswork (that I added).